plugins/orchestrator-backend/src/service/api/v2.test.ts

```diff
@@ -268,6 +310,342 @@ describe('getWorkflowById', () => {
+  });
+});
+
+describe('executeWorkflow', () => {
+  let mockSonataFlowService: SonataFlowService;
+  let mockDataIndexService: DataIndexService;
+  beforeEach(async () => {
+    jest.clearAllMocks();
+    // Mock SonataFlowService instance
+    mockSonataFlowService = createMockSonataFlowService();
+    mockDataIndexService = createMockDataIndexService();
+  });
+  it('executes a given workflow', async () => {
+    // Arrange
+    const workflowInfo = generateTestWorkflowInfo();
+    const execResponse = generateTestExecuteWorkflowResponse();
+    (mockDataIndexService.getWorkflowDefinition as jest.Mock).mockResolvedValue(
+      workflowInfo,
+    );
+
+    (mockSonataFlowService.executeWorkflow as jest.Mock).mockResolvedValue(
+      execResponse,
+    );
+    const workflowData = {
+      inputData: {
+        customAttrib: 'My customAttrib',
+      },
+    };
+    // Act
+    const actualResultV2: ExecuteWorkflowResponseDTO = await V2.executeWorkflow(
+      mockDataIndexService,
+      mockSonataFlowService,
+      workflowData,
+      workflowInfo.id,
+      'businessKey',
+    );
+
+    // Assert
+    expect(actualResultV2).toBeDefined();
+    expect(actualResultV2.id).toBeDefined();
+    expect(actualResultV2.id).toEqual(execResponse.id);
+    expect(actualResultV2).toEqual(execResponse);
+  });
+});
+
+describe('getInstances', () => {
+  let mockDataIndexService: DataIndexService;
+  const mockRequest: any = {
+    query: {},
+  };
+  const pagination = buildPagination(mockRequest);
+  beforeEach(() => {
+    jest.clearAllMocks();
+    mockDataIndexService = createMockDataIndexService();
+  });
+
+  it("Instance doesn't exists", async () => {
+    // Arrange
+
+    (mockDataIndexService.fetchProcessInstances as jest.Mock).mockRejectedValue(
+      new Error('No definition'),
+    );
+    // Act
+    const promise = V2.getInstances(mockDataIndexService, pagination);
+
+    // Assert
+    await expect(promise).rejects.toThrow('No definition');
+  });
+
+  it('1 items in process instance list', async () => {
+    const processInstance = generateProcessInstance(1);
+
+    (mockDataIndexService.fetchProcessInstances as jest.Mock).mockResolvedValue(
+      [processInstance],
+    );
+
+    // Act
+    const processInstanceV2: ProcessInstanceListResultDTO =
+      await V2.getInstances(mockDataIndexService, pagination);
+
+    // Assert
+    expect(processInstanceV2).toBeDefined();
+    expect(processInstanceV2.items).toBeDefined();
+    expect(processInstanceV2.items).toHaveLength(1);
+    expect(processInstanceV2.items?.[0].id).toEqual(processInstance.id);
+  });
+  it('10 items in process instance list', async () => {
+    const howmany = 10;
+    const processInstance = generateProcessInstances(howmany);
+
+    (mockDataIndexService.fetchProcessInstances as jest.Mock).mockResolvedValue(
+      processInstance,
+    );
+
+    // Act
+    const processInstanceV2: ProcessInstanceListResultDTO =
```

Comment:
```suggestion
    const processInstance: ProcessInstanceListResultDTO =
```
This file is for v2 tests. Do we need to use `v2` suffix here? Applicable to other places too.